src/App.js

```diff
@@ -1,17 +1,28 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
-export class App extends Component {
+export class App extends React.Component {
   state = {
     movies: moviesFromServer,
   };
 
-  addMovie = (movie) => {
-    // put your code here
-  };
+  submit = (values) => {
+    this.setState(state => ({
+      movies: [
+        ...state.movies,
+        {
+          title: values.title,
+          description: values.description,
+          imgUrl: values.imgUrl,
+          imdbUrl: values.imdbUrl,
+          imdbId: values.imdbId,
+        },
+      ],
+    }));
+  }
```

Comment:
это не сабмит, эта функция добавляет новый фильм
```suggestion
  addMovie = (movie) => {
    this.setState(state => ({
      movies: [
        ...state.movies,
        movie,
      ],
    }));
  }
```